.gitignore

```diff
@@ -36,6 +36,7 @@
 
 # Other
 *.vscode
+.vs/
```

Comment:
VSCode uses the `.vscode` directory to store its metadata. Are you sure this is actually used?

Reply:
Yup, it's storing various files like `VSWorkspaceState.json` and  `ProjectSettings.json`

Reply:
VSCode? Or Visual Studio? Because VSCode's metadata is stored in `.vscode`.